src/engineer/base.py

```diff
@@ -47,10 +48,49 @@ def engineer(
 
         self._process_dynamic(test_year, target_variable, pred_months, expected_length)
         if self.process_static:
-            self._process_static()
+            self._process_static(test_year=test_year)
```

Comment:
requires `test_year` so that can ignore test years in the calculation
of spatial means and global means of dynamic variables. 

These variables are now calculated from the original data.nc file